docs/admin/reference/removing_gpg_passphrase.rst

```diff
@@ -0,0 +1,27 @@
+Removing the Passphrase from a GPG Key
+======================================
+
+First, note your secret key's ID. You can find it by running the command:
+
+.. code-block:: sh
+
+  gpg --list-secret-keys --keyid-format=long
```

Comment:
Would need updating with a --homedir, and in general (tho not for this command) probably a --pinentry=loopback for any command with a passphrase prompt.